src/node.cc

```diff
@@ -303,6 +304,29 @@ MaybeLocal<Value> StartExecution(Environment* env, StartExecutionCallback cb) {
     first_argv = env->argv()[1];
   }
 
+#ifndef DISABLE_SINGLE_EXECUTABLE_APPLICATION
+  if (IsSingleExecutable()) {
```

Comment:
Give how specific this code is to SEA, it might be better to create a `node_sea.cc` and move SEA-related code there instead of letting them scatter around in the code base. It could also be a good idea to add a `sea` binding which contains a function that can be called from user land to get the code. Then we can just throw properties that are useful to SEA to that binding, without having to make the process object even more complex.

Reply:
@joyeecheung done, PTAL!